src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -697,6 +685,14 @@ public boolean isItemValidForSlot(final int index, final ItemStack stack)
     public int getField(final int id)
     {
         return 0;
+    }    /**
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ATrailingCommentCheck)